### DISCUSSION TITLE
Question about the look_at function

### DISCUSSION THREAD

**Discussion by JackYifan** [NVlabs/sionna#652] [Q&A]
Why the beta = theta-PI/2
![image](https://github.com/user-attachments/assets/3bc8539c-507d-4c5a-aaa6-ca37cd0138e5)

![image](https://github.com/user-attachments/assets/802dd0a3-9b23-46c9-85d0-52b596ef2811)

**Comment by merlinND** [comment]
Hello @JackYifan,

I believe this is just a consequence of the coordinate system conventions.
